File: code_18.py
def print_triangle(n):
    for i in range(1, n + 1):
        print("*" * i + 1)  # Logical mistake: adding 1 to the string.

# Main code
rows = 5
print_triangle(rows)  # Syntax mistake: missing parenthesis in function call.

# End message
print("Congratulations! You have qualified for the next round.")

File: cpp codes/code18.cpp
#include <iostream>
#include <vector>
using namespace std;

void team18() {
    vector<int> a = {10, 20, 30};
    vector<int> b = a + 5; // Logical Error: Cannot add a list and an integer
    for (int i = 0; i < b.size(); i++) {
        cout << b[i] << " ";
    }
    cout << endl;
    cout << "You qualify for the next round!" << endl;
}

int main() {
    team18();
    return 0;
}
